flake.nix

```diff
@@ -3,7 +3,7 @@
 
   inputs = {
     nixpkgs.follows = "holochain/nixpkgs";
-    versions.url = "github:holochain/holochain?dir=versions/0_2";
+    versions.url = "github:holochain/holochain?dir=versions/weekly";
```

Comment:
The `flake.lock` should have changed when this did, have you run `nix flake update`?

Reply:
Good catch, updated it